FireRender.Maya.Src/Context/TahoeContext.cpp

```diff
@@ -245,6 +246,39 @@ void TahoeContext::setupContext(const FireRenderGlobalsData& fireRenderGlobalsDa
 	checkStatus(frstatus);
 
 	updateTonemapping(fireRenderGlobalsData, disableWhiteBalance);
+
+	// OCIO
+	{
+		MStatus colorManagementStatus;
+		int isColorManagementOn = false;
+		colorManagementStatus = MGlobal::executeCommand(MString("colorManagementPrefs -q -cmEnabled;"), isColorManagementOn);
+
+		int isConfigFileEnable = false;
+		colorManagementStatus = MGlobal::executeCommand(MString("colorManagementPrefs -q -cmConfigFileEnabled;"), isConfigFileEnable);
+
+		if ((bool)isColorManagementOn && (bool)isConfigFileEnable)
```

Comment:
I would change it to "isColorManagementOn  > 0"  "isConfigFileEnable > 0" without (bool) casting